locales/en.json

```
@@ -74,5 +80,9 @@
 		"main": "`Main language`: `{{ langCode }}`",
 		"default": "`Default channel`: {{ defaultChannel }}",
 		"latency": "`Shard latency`: {{ shardLatency }}ms"
+	},
+	"moderator": {
+		"is": "{{ user }} is a moderator of the DISBOARD service.",
```

```suggestion
		"is": "{{ user }} is a moderator of DISBOARD.",
```

why more when few do trick

---

changed